app/assets/v2/js/grants/detail.js

```diff
@@ -3,12 +3,41 @@ const editableFields = [
   '#form--input__reference-url',
   '#contract_owner_address',
   '#grant-members',
-  '#amount_goal'
+  '#amount_goal',
+  '#category'
 ];
 
+const categoryOptions = [
```

Comment:
cc @frankchen07  @danlipert in case it changes how things are done with the media/core grants work being done in sprint now.